assets/js/index.js

```diff
@@ -44,4 +55,40 @@ function clearAddTodoInput() {
 function clearErrorMessage() {
   errorMessage.innerText = "";
   backendErrorMessage.innerText = "";
+}
+
+async function getTasks() {
+  try {
+    const response = await fetch(url);
+    const {data} = await response.json();
+    return data;
+  } catch {
+    backendErrorMessage.innerText = "Something went wrong!";
+  }
+}
+
+function changeDateFormat(createdAt) {
+  const dateTime = new Date(createdAt);
+  const year = dateTime.toLocaleDateString("en-US", { year: "numeric" });
+  const month = dateTime.toLocaleDateString("en-US", { month: "short" });
+  const day = dateTime.toLocaleDateString("en-US", { day: "numeric" });
+
+  return `${day} ${month} ${year}`;
+}
```

Comment:
Beautiful ❤️ 
You can change function name to something like `formatDate`.

Reply:
Also I recommend you to change the variable name `createdAt` to `date`. The reason is, the function can be generic.You can use this to format any given date like `updatedAt`, etc.

Reply:
Done.